### PR TITLE
Added 3 outputs to the ecs module, asked for by LAA. These output the…

### PR DESCRIPTION
**Pull request by ep-93**
… scale up / down policy.

### CODE REVIEW

**Review by dms1981 — COMMENTED** [review]
These all look like arn outputs, but the output names don't reflect that. Did you mean to just output arns, in which case, can you change the output names to reflect that?

File: outputs.tf [hunk 1]
```diff
@@ -22,3 +22,18 @@ output "ecs_task_execution_policy" {
...
+  description = "App scale up policy output"
+  value = aws_appautoscaling_policy.scaling_policy_up.arn
+}
+
+output "app_scale_down_policy"{
+  description = "App scale down policy output"
+  value = aws_appautoscaling_policy.scaling_policy_down.arn
+}
+
+output "app_autoscale_name"{
```
Comment (julialawrence) [inline outputs.tf]: Also this needs to be the name of the aws_autoscaling_group not the arn.

**Review by julialawrence — CHANGES_REQUESTED** [review]
Ditto on what David said as well.

File: outputs.tf (hunk 1 above)
Comment (julialawrence) [reply outputs.tf]: I mean line 36/37

**Review by julialawrence — DISMISSED** [review]
👍

**Review by julialawrence — APPROVED** [review]
Reapproving.

**Review by dms1981 — APPROVED** [review]
Looks good to me!

**Pull request merged**